src/helpers.ts

```diff
@@ -72,7 +72,7 @@ export function splitCells(tableRow: string, count?: number) {
   if (!cells[0].trim()) {
     cells.shift();
   }
-  if (cells.length > 0 && !cells[cells.length - 1].trim()) {
+  if (!cells.at(-1)?.trim()) {
```

Comment:
This looks like a bug.

Consider the case when `cells = []`.

```
console.log(cells.length > 0 && !cells[cells.length - 1].trim()); // false
console.log(!cells.at(-1)?.trim()) // true
```

Reply:
Thanks for the catch. Since all the unit tests still pass, I wonder if for some reason cells.length always happens to be > 0  on this line. I'll fix it though.